demos/_unit-test/console.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
why the total coverage decreased by -2%?

Reply:
Is Behat part of the coverage? I move some tests to Behat but it does not seem to take it into account.

Reply:
yes, it is related with Wizard etc., so no core code, seems ok so then